packer/templates/template-ubuntu-14.04-packages-noatlas.json

```diff
@@ -0,0 +1,95 @@
+{
+    "push": {
+      "name": "",
+      "vcs": true
+    },
+    "variables": {
+    },
+    "provisioners": [
+        {
+            "type": "shell",
+            "script": "scripts/base.sh",
+            "execute_command": "echo 'vagrant'|sudo -S bash '{{.Path}}'"
+        },
+        {
+            "type": "shell",
+            "script": "scripts/virtualbox.sh",
+            "execute_command": "echo 'vagrant'|sudo -S bash '{{.Path}}'",
+            "only": ["virtualbox-iso"]
+        },
+        {
+            "type": "shell",
+            "scripts": [
+                "scripts/vagrant.sh",
+                "scripts/dep.sh"
+            ],
+            "execute_command": "echo 'vagrant'|sudo -S bash '{{.Path}}'"
+        },
+        {
+            "type": "ansible-local",
+            "inventory_groups": "local",
+            "playbook_file": "ansible/rackhd_package.yml",
+            "playbook_dir": "ansible"
+        },
+        {
+            "type": "shell",
+            "scripts": [
+                "scripts/firstuser.sh",
+                "scripts/cleanup.sh",
+                "scripts/zerodisk.sh"
+            ],
+            "execute_command": "echo 'vagrant'|sudo -S bash '{{.Path}}'"
+        }
+    ],
+  "builders": [
+    {
+      "type": "virtualbox-iso",
+      "boot_command": [
+        "<esc><wait>",
+        "<esc><wait>",
+        "<enter><wait>",
+        "/install/vmlinuz<wait>",
+        " auto<wait>",
+        " console-setup/ask_detect=false<wait>",
+        " console-setup/layoutcode=us<wait>",
+        " console-setup/modelcode=pc105<wait>",
+        " debian-installer=en_US<wait>",
+        " fb=false<wait>",
+        " initrd=/install/initrd.gz<wait>",
+        " kbd-chooser/method=us<wait>",
+        " keyboard-configuration/layout=USA<wait>",
+        " keyboard-configuration/variant=USA<wait>",
+        " locale=en_US<wait>",
+        " netcfg/get_hostname=rackhd-demo<wait>",
+        " noapic<wait>",
+        " interface=auto",
+        " preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed.cfg<wait>",
+        " -- <wait>",
+        "<enter><wait>"
+      ],
+      "boot_wait": "10s",
+      "disk_size": 40960,
```

Comment:
Do we want "headless": "true" ?
otherwise, user have to run packer build in GUI , in that case, document should reflect that.